pkg/api/v1/types.go

```diff
@@ -317,6 +317,8 @@ type VirtualMachineInstanceNetworkInterface struct {
 	IP string `json:"ipAddress,omitempty"`
 	// Hardware address of a Virtual Machine interface
 	MAC string `json:"mac,omitempty"`
+	// Name of the interface, corresponds to the pod bridge the interface is
+	Name string `json:"name,omitempty"`
```

Comment:
Does not matter much since the status is only manipulated by the controller, but since the `Name` is mandatory now, we should remove the `omitempty`.